.github/workflows/CI.yml

```diff
@@ -47,6 +47,22 @@ jobs:
         with:
           NO_COVERAGE: ${{ matrix.NO_COVERAGE }}
 
+  test-cygwin:
+    name: "cygwin64 - GAP master - HPCGAP no"
+    if: ${{ !(github.event_name == 'pull_request' && github.event.pull_request.head.repo.full_name == github.repository) }}
+    runs-on: windows-latest
+    steps:
+      - name: "Set git to use UNIX-style line endings"
+        run: |
+          git config --global core.autocrlf false
+          git config --global core.eol lf
+      - uses: actions/checkout@v2
+      - uses: gap-actions/setup-cygwin-for-gap@v1
+      - uses: wilfwilson/setup-gap-for-packages@cygwin
+        with:
+          GAP_PKGS_TO_BUILD: ''
+      - uses: wilfwilson/run-test-for-packages@cygwin
```

Comment:
Are these two action variants still needed? If so, what do they change?

Reply:
They change the shell in which the command is run, and they change the way in which the command is invoked. This should be reconcilable, but I've not worked it out yet. I'll open PRs to the relevant GAP actions so that you can see the diff.

Reply:
@fingolfin https://github.com/gap-actions/build-pkg/pull/1/files this is a good example of the kind of change that I made to make the actions work in the Windows/Cygwin environment. In particular, it uses a different shell (`cmd` rather than `bash`), and within that shell it calls a few commands before calling Cygwin's version of bash in a particular way.

I don't know how much of this is necessary.

Reply:
I'll ask @ChrisJefferson about it when I next talk to him.